src/main.rs

```diff
@@ -114,7 +119,7 @@ where
     let raw_mode = opts.raw;
 
     let network_utilization = Arc::new(Mutex::new(Utilization::new()));
-    let ui = Arc::new(Mutex::new(Ui::new(terminal_backend)));
+    let ui = Arc::new(Mutex::new(Ui::new(terminal_backend, opts)));
```

Comment:
What do you think about making this explicit and passing a struct that would look something like this?
```rust
struct TablesToRender {
    processes: bool,
    connections: bool,
    addresses: bool,
}
```

Reply:
It's an option but I can also see how we might add more rendering options in the future. So I would rather have a `RenderOpts` struct, but they concept is the same I guess.